### ISSUE TITLE
Peer discovery issues

### ISSUE THREAD

**Issue by ch4r10t33r** [eth-infinitism/bundler-spec-tests#62]
**Voltaire** bundlers are unable to discover Silius peers on the network.

**Comment by Vid201** [comment]
It would be helpful to have a test for peer connections. We could introduce a new debug endpoint `debug_bundler_peers` to check peers of the bundler in the test.

**Comment by sherifahmed990** [comment]
Description for the issue:

Voltaire can discover peers when connected to a Skandha boot node "enr:-J24QG7VfzC_pWTNhbMk67-rujhFQ1xTYG3eEwb8IgH2qwESfodulOz0iPYChAoK3zaa
Kw0lf4dcvSX3Cp7CRBF3Jm-UdiGNoYWluX2lkiKc2qgAAAAAAgmlkgnY0gmlwhBKpK9OJc2VjcDI1NmsxoQMxl3O4Gr_4mZJsfkNN
HQERjURl1G-GhoyKR0NWtEK9vkoN0Y3CCEPGDdWRwghDx"

Voltaire can't discover any peers when connecting to a Silius boot node "enr:-J24QFb-3hQjofsf-BT_DphduK4QAodRn4bJP-JriJVQCTfKC73qc2Nc3s8niwRb6C5jjG8GG9DsHpfK2sLyVOhUzoQBiGNN
oYWluX2lkiKc2qgAAAAAAgmlkgnY0gmlwhII9OBqJc2VjcDI1NmsxoQKkD8kVT-qEJjEaHzokEhqhe5QpStwq0LAa5QsOYk_5foNN
0Y3CCEPGDdWRwghDx"

**Comment by shahafn** [comment]
@ch4r10t33r  is this issue specific to the test?
@Vid201 you suggested adding a new debug endpoint, should it be part of the spec test for all bundlers? Sounds like an issue worth bringing up in the next call before we introduce it here, no?

**Comment by Vid201** [comment]
> @ch4r10t33r is this issue specific to the test? @Vid201 you suggested adding a new debug endpoint, should it be part of the spec test for all bundlers? Sounds like an issue worth bringing up in the next call before we introduce it here, no?

Yes, I think the new endpoint could be helpful for p2p tests and should be part of spec test - we could check if peers are successfully connected to each other.

Happy to discuss that on the next call.
